infrastructure/main.tf

```diff
@@ -43,3 +47,11 @@ module "aks" {
   resource_group_name = module.resource_group.resource_group_name
   depends_on          = [module.resource_group]
 }
+
+module "traffic_manager" {
+  source              = "./modules/resource_templates/traffic_manager"
+  name                = local.traffic_manager_name
+  resource_group_name = module.resource_group.resource_group_name
+  public_ip_name      = local.public_ip_name
+  depends_on          = [module.resource_group]
```

Comment:
Just a thought, but would it be useful making this module dependent on the AKS module being deployed? that way we could tie the two together and ensuring a traffic manager would only be there if the AKS cluster is?